testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -10901,6 +10901,26 @@ void checkEndsWith(SqlOperatorFixture f0, FunctionAlias functionAlias) {
             + "requires extra delimiter argument", false);
   }
 
+  /** Tests the {@code SPLIT_PART} operator. */
+  @Test void testSplitPartFunction() {
+    final SqlOperatorFixture f0 = fixture().setFor(SqlLibraryOperators.SPLIT_PART);
+    f0.checkFails("^split_part('hello', 1)^",
+        "No match found for function signature SPLIT_PART\\(<CHARACTER>, <NUMERIC>\\)",
+        false);
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.POSTGRESQL);
+
+    f.checkScalar("SPLIT_PART('abc~@~def~@~ghi', '~@~', 2)", "def", "VARCHAR(15) NOT NULL");
+    f.checkScalar("SPLIT_PART('abc,def,ghi,jkl', ',', 3)", "ghi", "VARCHAR(15) NOT NULL");
+
+    f.checkScalar("SPLIT_PART('abc~@~def~@~ghi', '~@~', -1)", "ghi", "VARCHAR(15) NOT NULL");
+    f.checkScalar("SPLIT_PART('abc,def,ghi,jkl', ',', -2)", "ghi", "VARCHAR(15) NOT NULL");
+
+    f.checkScalar("SPLIT_PART('h,e,l,l,o', ',', 7)", "", "VARCHAR(9) NOT NULL");
+    f.checkScalar("SPLIT_PART('h,e,l,l,o', ',', -7)", "", "VARCHAR(9) NOT NULL");
+
+    f.checkScalar("SPLIT_PART('abc,,ghi', ',', 2)", "", "VARCHAR(8) NOT NULL");
```

Comment:
Add unit test to cover NULL parameter.

Reply:
Addressed in latest PR, thanks!

Reply:
I noticed you did not covered the `SPLIT_PART('abc,,ghi', ',', null)`

Reply:
Added now the third paras null case, thanks!